charts/hybridnet/templates/deployments.yaml

```diff
@@ -123,3 +123,109 @@ spec:
           ports:
             - containerPort: 9898
               name: webhook-port
+
+{{ if and .Values.typha .Values.daemon.enableFelixPolicy }}
```

Comment:
`.Values.typha` is struct, so should `.Values.typha.enabled` be introduced?

Reply:
I think maybe `--set typha=null` is enough for this, because `typha` struct should always to be checked in case it's empty.